utils/check_copies.py

```diff
@@ -298,6 +315,41 @@ def _rep_link(match):
     return "\n".join(result)
 
 
+def convert_to_localized_md(model_list, localized_model_list, format_str):
+    """Convert `model_list` to each localized README."""
+
+    def _rep(match):
+        title, model_link, paper_institutions, paper_title_link, paper_authors, supplements = match.groups()
+        return format_str.format(
+            title=title,
+            model_link=model_link,
+            paper_institutions=paper_institutions,
+            paper_title_link=paper_title_link,
+            paper_authors=paper_authors,
+            supplements=" " + supplements.strip() if len(supplements) != 0 else "",
+        )
+
+    _re = re.compile(
```

Comment:
Let's give this a name a bit longer that has some meaning, and add a comment and what pattern this re catches for readability.